hera_pspec/pspecdata.py

```diff
@@ -343,6 +347,8 @@ def clear_cache(self, keys=None):
                 except(KeyError): pass
                 try: del(self._iC[k])
                 except(KeyError): pass
+                try: del(self._RegFactor[k])
+                except(KeyError): pass
```

Comment:
you probably also want to clear `self.r_params` here too

Reply:
Now clears r_params.